frontend/src/routes/(root)/(logged)/workspace_settings/+page.svelte

```diff
@@ -587,6 +598,22 @@
 			timeout: 5000
 		})
 	}
+
+	async function editCriticalAlertMuteSetting() {
+		await SettingService.workspaceMuteCriticalAlertsUi({
+			workspace: $workspaceStore!,
+			requestBody: {
+				mute_critical_alerts: criticalAlertUIMuted
+			}
+		})
+		sendUserToast(
+			`Critical alert UI mute setting for workspace is set to ${criticalAlertUIMuted}\nreloading page...`
+		)
+		// reload page after change of setting
+		setTimeout(() => {
+			window.location.reload()
+		}, 3000)
```

Comment:
no need to reload the page, just say that they need to reload in the toast, this is too harsh